src/plugins/dashboard/common/types.ts

```diff
@@ -85,3 +86,34 @@ export type SavedDashboardPanel730ToLatest = Pick<
   readonly id?: string;
   readonly type: string;
 };
+
+export interface DashboardListSource {
```

Comment:
Why do these types need to be defined in `common`? Seems like they are only used in `public`, not server-side.

Reply:
+1. The types `DashboardListProviderFn` and `DashboardDisplay` are also duplicated in the `src/plugins/dashboard/public/application/legacy_app.ts` file. You can consolidate them into the same types file.

Reply:
I agree.  Fixed